### ISSUE TITLE
Task scheduling takes too long

### ISSUE THREAD

**Issue by IAmSalvaMartini** [cirruslabs/cirrus-ci-docs#827]
I'm  on the community cluster and really love it, and as per our previous discussion, I've managed to fine tune my build to limit it to less than 2 hours by increasing ram but now I can't seem to get scheduled for over 5 hours. I don't know if this is something you might be already aware of.If it is, you can go ahead and close this and sorry for the inconvienience caused. Also, thank you for this awesome software

**Comment by fkorotkov** [comment]
Could you please provide a link to a pending task?

**Comment by IAmSalvaMartini** [comment]
> Could you please provide a link to a task you are referring to?

I ended up cancelling it, but here you go https://cirrus-ci.com/task/4733861885116416

**Comment by fkorotkov** [comment]
Yes, this is all because there are too many folks like you trying to use 16 CPU containers to build AOSP. This is starting affecting regular users of Cirrus CI, their tasks with high CPUs requirements also experience issues scheduling. We have to put an end to such exploitation of the free tier. Using Cirrus CI as a build farm is not intended. Free tier is used by OSS projects for testing and validating integrations and not as a raw powerhouse for building custom Android ROMs.

As I mentioned in https://github.com/cirruslabs/cirrus-ci-docs/issues/825#issuecomment-808985127, there are options to use Cirrus CI without impacting everyone on the free tier.
